lib/Runtime/Base/Utf8SourceInfo.cpp

```diff
@@ -414,9 +414,11 @@ namespace Js
 #ifdef ENABLE_SCRIPT_DEBUGGING
         if (this->HasDebugDocument() && this->GetDebugDocument()->HasDocumentText())
         {
+#ifdef _WIN32
             // ToDo (SaAgarwa): Fix for JsRT debugging
             IDebugDocumentText *documentText = static_cast<IDebugDocumentText *>(this->GetDebugDocument()->GetDocumentText());
             if (documentText->GetName(DOCUMENTNAMETYPE_URL, sourceName) == S_OK)
+#endif
             {
                 return true;
             }
```

Comment:
Whole if should be under #ifdef _WN32, we don't want to retrun true without setting sourceName. and in JsRT there is no sourceName support.